tests/msg_queue_print/tests/01-run.py

```diff
@@ -8,17 +8,21 @@
 
 # @author      Julian Holzwarth <julian.holzwarth@fu-berlin.de>
 
+import os
 import sys
 from testrunner import run
 
 
 def testfunc(child):
     child.expect("No message queue")
-    child.expect("Message queue of thread 2")
+    child.expect(r"Message queue of thread \d+")
```

Comment:
don't expect wildcard at end of string, it might match partially. (enclose it with something)

Reply:
`\d+` would match the 1 of 12 with pexpect. in this case, we're not expecting double digit thread ids, but then do just `\d` to be explicit, or fix to read `(\d+)\r\n'`

https://github.com/kaspar030/RIOT/blob/5d7477fa5f240e7989acb1158c8342d06b9c8360/tests/README.md#L67-L94

Reply:
I don't think the actual number of the thread is important here, so I don't think it is important to match exactly here. Can we agree to get this in without your suggestion?

Reply:
Addressed